benchmark/bootstrap_panoptes.rb

```diff
@@ -33,45 +33,53 @@ def create_db_schema
     @pg.exec <<-SQL
```

Comment:
`DATA_DIR = 'tmp'`

This should either create a local tmp directory or use `DATA_DIR = '/tmp'`